annaeliana7/Healthy-Kitchen
# Feature

## What
I have created a new webpage that is mobile responsive, and includes the following technologies:

- Linters (ESLint and Stylelint) to ensure code quality and consistency
- Webpack for module bundling and asset management
- Bootstrap for a responsive grid system and pre-designed components
- Javascript for interactive functionality
- HTML and CSS for the webpage structure and styling
- I have also included a README file that explains how to run the webpage locally, and how to make changes to it

## Why
Please review the code and let me know if there are any issues or suggestions.

## How
Steps to test the webpage locally:

- Clone the repository
- Run npm install to install all the dependencies
- Run npm run start to start the development server
- Open http://localhost:8080 in your browser to see the webpage

Thank you!

Note: please make sure that you have installed webpack, npm and node.js on your computer.

## Project Demos
![project_demo](https://user-images.githubusercontent.com/112557951/214791748-e4b0351a-6e9f-4a8f-a3d0-66df718e8218.jpeg)
![project_demo-2](https://user-images.githubusercontent.com/112557951/214791786-9cb9ff06-7830-4741-88f7-aed979e1f98c.jpeg)


### Changes details
- Detail one
- Detail two

## Missed anything?
- [ ] Explained the purpose of this PR.
- [ ] Self reviewed the PR.
- [ ] Added or updated test cases.
